.gitmodules

```diff
@@ -19,3 +19,6 @@
 [submodule "third_party/cpp-btree"]
 	path = third_party/cpp-btree
 	url = https://github.com/algorithm-ninja/cpp-btree
+[submodule "third_party/cqf"]
+	path = third_party/cqf
```

Comment:
Once you mention the original repo, once yours.
Any reason to not use the original one?

Reply:
The original one uses flexible arrays for runtime configurated filters in its headers. I can't compile that in C++. Mine uses a set of predefined configurations.